Makefile

```diff
@@ -254,7 +255,8 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool podman --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	# FIXME: hardcoded bundle below should use go.mod pinned version for manila bundle
+	$(OPM) index add --container-tool podman --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS),quay.io/openstack-k8s-operators/manila-operator-bundle:latest $(FROM_INDEX_OPT)
```

Comment:
Same comment of the previous one here: long term we might want to call a script where the bundles listed in the `dependencies.yaml` are concatenated in the command, but +1 to the hardcoded manila for now.